README.md

```diff
@@ -1041,6 +1041,50 @@ _You can enable the following settings in Xcode by running [this script](resourc
   }
   ```
 
+* <a id='doc-comments'></a>(<a href='#doc-comments'>link</a>) Use doc comments (`///`) before declarations, and regular comments (`//`) elsewhere. [![SwiftFormat: docComments](https://img.shields.io/badge/SwiftFormat-docComments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#docComments)
+
+  <details>
+
+  ```swift
+  // WRONG
+
+  // A planet that exists somewhere in the universe.
+  class Planet {
```

Comment:
```suggestion
  final class Planet {
```

Reply:
Since we say to prefer `final` this seems like the right way to write this example?